src/Accordion/Accordion.spec.js

```diff
@@ -0,0 +1,115 @@
+import React from 'react';
+import { createUniDriverFactory } from 'wix-ui-test-utils/uni-driver-factory';
+
+import Accordion from './Accordion';
+import { accordionPrivateDriverFactory } from './Accordion.private.driver';
+
+import { eventually } from '../../test/utils/unit/eventually';
+import { BUTTON_TYPES } from './constants';
+
+describe('Accordion', () => {
+  const FakeIcon = () => <div>fake icon</div>;
+  const createDriver = createUniDriverFactory(accordionPrivateDriverFactory);
+
+  describe('items data', async () => {
+    it('should render a list of items', async () => {
+      const data = [
+        {
+          title: 'first item',
+        },
+      ];
+      const driver = createDriver(<Accordion data={data} />);
+      expect(await driver.getItemTitleAt(0)).toEqual('first item');
+    });
+
+    it('should not render any items', async () => {
+      const driver = createDriver(<Accordion />);
+      expect(await driver.getAmountOfItems()).toEqual(0);
+    });
+
+    it('should render item with icon', async () => {
+      const data = [
+        {
+          title: 'first item',
+          icon: <FakeIcon />,
+          content: 'first item content',
+        },
+      ];
+      const driver = createDriver(<Accordion data={data} />);
+      expect(await driver.isIconExistsAt(0)).toBeTruthy();
```

Comment:
please be explicit throughout this file. `toBeTruthy` and `toBeFalsy` can be false positive
```suggestion
      expect(await driver.isIconExistsAt(0)).toBe(true);
```

Reply:
Could you clarify on this one? I'm not really aware that it can be false positive and I see it's being used in other files in wsr.